dev/tests/acceptance/tests/functional/Magento/FunctionalTest/ConfigurableProductWishlist/ActionGroup/StorefrontCustomerCheckConfigurableProductInWishlistActionGroup.xml

```diff
@@ -24,20 +24,4 @@
         <seeElement selector="{{StorefrontCustomerWishlistProductSection.ProductAddToCartByName(productVar.name)}}" stepKey="AssertWishlistAddToCart"/>
         <seeElement selector="{{StorefrontCustomerWishlistProductSection.ProductImageByName(productVar.name)}}" stepKey="AssertWishlistProductImage"/>
     </actionGroup>
-
-    <!-- Check configurable product in wishlist sidebar -->
-    <actionGroup name="StorefrontCustomerCheckConfigurableProductInWishlistSidebar">
```

Comment:
Shouldn't we keep this action group in a separate file, but marked as deprecated for backward compatibility?
@lbajsarowicz, what do you think?

Reply:
We should, actually.